public/data/Narendrapawar.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Narendrapawar",
+  "type": "personal",
+  "bio": "I am a frontend developer and an open-source enthusiast.",
+  "avatar": "https://github.com/Narendrapawar11.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Narendrapawar11",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Narendrapawar11",
```

Comment:
**You've placed a link to a suspended account here.** 🚫

You're required to use the link of your **active account** or remove the twitter section from here in case you don't have one. 
![image](https://user-images.githubusercontent.com/88786642/195985610-9af4893f-8d8f-44fb-8f92-0cdfa5046c91.png)